Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3596,11 +3600,22 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
 
 id UnifiedPDFPlugin::accessibilityObject() const
 {
-#if PLATFORM(MAC)
     return m_accessibilityDocumentObject.get();
-#endif
+}
+
+#if !PLATFORM(MAC)
+id UnifiedPDFPlugin::accessibilityHitTestInPage(WebCore::FloatPoint point)
+{
+    RefPtr pageRoot = this->page();
+    if (!pageRoot)
+        return nil;
+
+    auto [page, pointInPage] = rootViewToPage(pageRoot->chrome().screenToRootView(WebCore::IntPoint(point)));
+    if (page && [page respondsToSelector:@selector(accessibilityHitTest:withPlugin:)])
```

Comment:
Do we need to nil-check `page`? Won't `respondsToSelector` return `NO` if `page` is nil?